Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -294,6 +298,31 @@ void UpdatePadding()
 
 			_lastSizeRequest = null;
 		}
+		
+		void UpdateTextType()
+		{
+			if (Element.Text == null)
+				return;
+
+			switch (Element.TextType)
+			{
+
+				case TextType.Html:
+					if (Forms.IsNougatOrNewer)
+						Control.SetText(Html.FromHtml(Element.Text, FromHtmlOptions.ModeCompact), TextView.BufferType.Spannable);
```

Comment:
- Can this code be centralized into FormsTextView so it's not duplicated between the fast and slow renderer?
- Also, I don't think setting the text here is correct. Shouldn't this just call *UpdateText*? And then inside UpdateText you figure out whether the user is wanting to set HTML or PlainText? The way it's currently implemented if the user sets Element.Text it will always set the text as if TextType == PlainText

```C#
        void Button_Clicked(object sender, EventArgs e)
        {
            lblTextChanges.TextType = TextType.Html;
            lblTextChanges.Text = "<b>" + DateTime.Now.ToString() + "</b>";
        }
```

The above code will not display as HTML but it should